### PR TITLE
chore(EpicTesting): Fix issue in github testing.

### PR DESCRIPTION
**Pull request by jdetle**
I checked codecov and some of the tests weren't actually working. This should hopefully fix things!(?)

### CODE REVIEW

**Comment by jdetle** [comment]
Okay, shoot, I found some problems with the work I did.

File: src/notebook/epics/github-publish.js [hunk 1]
```diff
@@ -156,14 +156,14 @@ export function handleGistError(store, error) {
...
-      message: githubError.message,
+      message: error.message,
       level: 'error',
     });
     return;
   }
   notificationSystem.addNotification({
     title: 'Unknown Publication Error',
-    message: error.toString(),
+    message: err.toString(),
```
Comment (rgbkrk) [inline src/notebook/epics/github-publish.js]: Why `err` here?

Comment (jdetle) [reply src/notebook/epics/github-publish.js]: Good question, I'm not sure what we would want to do if the error message cant be parsed. But this was due to the fact that I was passing a message that couldn't be parsed.

File: src/notebook/epics/github-publish.js [hunk 2]
```diff
@@ -139,13 +139,13 @@ export function publishNotebookObservable(github, notebook, filepath,
  * @param  {Immutable.Record} store - Redux store containing current state.
  *
  */
-export function handleGistError(store, error) {
+export function handleGistError(store, err) {
   const state = store.getState();
   const notificationSystem = state.app.get('notificationSystem');
+  const error = JSON.parse(err);
```
Comment (rgbkrk) [inline src/notebook/epics/github-publish.js]: Since this comes from a remote, I'd wrap this in a try catch block.

**Pull request merged**